lib/opal/cli_runners/node_modules/.bin/chrome-remote-interface.cmd

```diff
@@ -0,0 +1,17 @@
+@ECHO off
```

Comment:
can we have node_modules updates in a separate commit? would make a review much simpler

Reply:
I made 3 commits